hydra-configs-torchvision/tests/test_instantiate_models.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import pytest
+from hydra.utils import get_class, instantiate
+from omegaconf import OmegaConf
+
+import torchvision.models as models
```

Comment:
I might suggest tidying up the imports here slightly. Not the end of the world, but I think it would be better to follow PEP8:

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

>Imports should be grouped in the following order:
>    1. Standard library imports.
>    2. Related third party imports.
>    3. Local application/library specific imports.

>You should put a blank line between each group of imports.

I'm probably also guilty of this in some of the other tests, but I want to continue improving the standards.

Reply:
I suggest to integrate with isort.